src/main/java/gregtech/api/capability/IDualHandler.java

```diff
@@ -1,14 +1,15 @@
 package gregtech.api.capability;
 
 import net.minecraftforge.items.IItemHandler;
+import net.minecraftforge.items.IItemHandlerModifiable;
 
-public interface IDualHandler {
+public interface IDualHandler extends IItemHandlerModifiable, IMultipleTankHandler {
```

Comment:
Do we actually still need this interface, now that Quantum Storage has been switched to use DualHandler? Can't we remove this interface and put everything in DualHandler directly, or is there some reason to keep it around?